### PR TITLE
Feat/save game score

### LINKED ISSUE #51: Save game score
# :white_check_mark: What's needed
Save the score at the end of the game

## 📋 Subtasks
- [x] save score
- [x] save username

### PR DESCRIPTION
**Pull request by mathieurobert1**
Save the score in files

More details : #51

### CODE REVIEW

File: src/Core/Core.cpp [hunk 1]
```diff
@@ -32,13 +33,30 @@ void Arcade::Core::start(const std::string & graphicLib)
...
+{
+    std::string prefix = "./lib/arcade_";
+    std::string suffix = ".so";
+
+    if (path.length() < 17)
+        return false;
+    if (path.compare(path.length() - suffix.length(), suffix.length(), suffix) == 0 &&
+        path.compare(0, prefix.length(), prefix) == 0)
+        return true;
+    std::cout << "nop" << std::endl;
```
Comment (Thomaltarix) [inline src/Core/Core.cpp]: Hello ?

File: src/Core/Saveinfo.cpp [hunk 2]
```diff
@@ -9,33 +9,41 @@
...
+    if (!std::getline(stream, line)) {
+        stream.close();
         return false;
-    tmp.str(line1);
-    if(!(tmp >> score))
-        return false;
-    std::getline(stream, username);
+    }
+    score = atoi(line.c_str());
+    std::cout << score << std::endl;
```
Comment (Thomaltarix) [inline src/Core/Saveinfo.cpp]: Is that debug ?

**Review by Thomaltarix — CHANGES_REQUESTED** [review]
You just missed some debug, but overall, that's pretty great

File: src/Core/Core.cpp (hunk 1 above)
Comment (mathieurobert1) [reply src/Core/Core.cpp]: oups

File: src/Core/Saveinfo.cpp (hunk 2 above)
Comment (mathieurobert1) [reply src/Core/Saveinfo.cpp]: oups

**Review by Thomaltarix — APPROVED** [review]
Clean

**Pull request merged**